modules/Classroom/components/MessageList/MessageList.tsx

```diff
@@ -20,7 +55,7 @@ const MessageList: React.FC<IMessageListProps> = ({ classroomId }) => {
   }
 
   const sortedMessages = [...messages].sort(
-    (a, b) => dayjs(b.createdAt).unix() - dayjs(a.createdAt).unix(),
+    (a, b) => new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime(),
```

Comment:
I think this can be abstracted out into a generalized shared method taking the array, sortByField and order (i.e. asc or desc) and then call the method here